src/PublicApiGenerator/CodeNormalizer.cs

```diff
@@ -19,6 +20,8 @@ internal static class CodeNormalizer
         internal const string StaticMarker = "static_C91E2709_C00B-4CAB_8BBC_B2B11DC75E50 ";
         internal const string ReadonlyMarker = "readonly_79D3ED2A_0B60_4C3B_8432_941FE471A38B ";
         internal const string AttributeMarker = "_attribute_292C96C3_C42E_4C07_BEED_73F5DAA0A6DF_";
+        internal const string EventModifierMarkerTemplate = "_{0}_292C96C3C42E4C07BEED73F5DAA0A6DF_";
+        internal const string EventRemovePublicMarker = "removepublic";
```

Comment:
Any reason not to use a GUID to make collisions even less likely?

Reply:
The marker becomes part of the `_{0}_GUID` signature which is matched. So it is already unique enough